sapi/phpdbg/phpdbg.c

```diff
@@ -1076,7 +1076,7 @@ void phpdbg_sighup_handler(int sig) /* {{{ */
 } /* }}} */
 #endif
 
-void *phpdbg_malloc_wrapper(size_t size ZEND_FILE_LINE_DC ZEND_FILE_LINE_ORIG_DC) /* {{{ */
+ZEND_ATTRIBUTE_MALLOC ZEND_ATTRIBUTE_ALLOC_SIZE(1) void *phpdbg_malloc_wrapper(size_t size ZEND_FILE_LINE_DC ZEND_FILE_LINE_ORIG_DC) /* {{{ */
```

Comment:
Nit: Current code always puts `ZEND_ATTRIBUTE_ALLOC_SIZE` at the end.

Reply:
that breaks the build locally.

Reply:
Ah, so interestingly for forward declarations (which is the only place this attribute is used so far) order doesn't matter. This looks good then.